### ISSUE TITLE
Custom Root CA

### ISSUE THREAD

**Issue by petercipov** [SeleniumHQ/docker-selenium#1176]
## 🚀 Feature Proposal

Add custom root CA certificate to all selenium browser docker images. Provide that root ca in your github repo.

## Motivation

need to record all network calls of browser to track down random network error of infrastructure I am testing. 

## Example
selenium (Using Proxy setting of remote webdriver) -> hub -> nodechrome -> **mitmproxy** -> infrastructure.

mitm proxy will generate HAR file with all network history

Backend I am testing is using websockets and webrtc so requires valid ssl origin to initialize. setting chrome ignore invalid certificates does not work for me.

**Comment by bikashkarmakar** [comment]
Many of our internal sites which use the organisation self-signed root certificate and we are unable to bypass SSL: CERTIFICATE_VERIFY_FAILED error. Would be of great help for me.

**Comment by diemol** [comment]
Thanks for the suggestion, @petercipov. However, this is (for now) out of the scope of this project. Our intention is to provide generic Docker images for Selenium Grid that can be further extended, and this is a good example of what a user could do with them.

Adding a custom CA could disturb the current experience for many users, as this would need to be embedded in the container at build time to be efficient, not at runtime. 

By no means this is a hard "NO", maybe in the future the use case is more popular and it can be added without affecting the users experience. An example of that is video recording, a couple of years ago we did not want to add it, but today it is a common use case, and we have added it.

Steps to add your own certificate can be seen in many posts around, like this [one](https://stackoverflow.com/questions/42292444/how-do-i-add-a-ca-root-certificate-inside-a-docker-image). 

I will close this for now, but we are always happy to revisit this in case things change.
